contrib/example-backend/controllers/clusterbinding/clusterbinding_reconcile.go

```diff
@@ -217,3 +224,43 @@ func (r *reconciler) ensureRBACClusterRoleBinding(ctx context.Context, clusterBi
 
 	return nil
 }
+
+func (r *reconciler) ensureRBACRoleBinding(ctx context.Context, clusterBinding *kubebindv1alpha1.ClusterBinding) error {
+	binding, err := r.getRoleBinding("kube-binder", clusterBinding.Namespace)
+	if err != nil && !errors.IsNotFound(err) {
+		return fmt.Errorf("failed to get RoleBinding \"kube-bind\": %w", err)
+	}
+
+	expected := &rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "kube-binder",
```

Comment:
does this equal the service-account name?

Reply:
nack, it references the kubeconfigsecretref.name with the name of the SA.

Do we want to have it the same as well? sgtm actually